src/assessments/seo/KeyphraseDistributionAssessment.js

```diff
@@ -32,8 +30,8 @@ class KeyphraseDistributionAssessment extends Assessment {
 
 		const defaultConfig = {
 			parameters: {
-				goodDistributionScore: 0.4,
-				acceptableDistributionScore: 0.6,
+				goodDistributionScore: 30,
```

Comment:
In the current version this is set to `40`. I think it might be a good idea to keep this 40 in order to keep continuity in the boundaries.